src/content/quickstart/kind/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2020-03-17T13:36:18+01:00
+title: "KIND"
+---
+
+## KIND
+
+[KIND](https://github.com/kubernetes-sigs/kind) is a tool to run local Kubernetes clusters inside Docker container nodes.
```

Comment:
Janki, the guide you're writing is may be more appropiate for the development side.

A quickstart with kind for final users, IMO, should write:

1) how to deploy the kind clusters manually (copy & paste instructions)
2) how to use subctl manually to deploy the latest release version.

It should be like the other quikckstarts, but with kind, just doing kind with their default CNI, no weave or anything like that.

Reply:
See this one as an example

Reply:
https://submariner-io.github.io/en/quickstart/openshiftsd/